AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptionStatusType.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TranscriptionStatusType.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@objc public enum TranscriptionStatusType: Int, Equatable, CustomStringConvertible {
+    case unknown = 0
```

Comment:
Just curious, why do we need the int value?

Reply:
The `@objc` modifier on enum in Swift requires an Int type